Extensions/TileMap/tilemapruntimeobject-pixi-renderer.ts

```diff
@@ -55,11 +59,28 @@ namespace gdjs {
         this._pixiObject,
         tileMap,
         textureCache,
+        // @ts-ignore
```

Comment:
why the ignore everywhere?

Reply:
_displayMode is typed as string in the object but as an enum in the method. I could force the type in the object